libdnf/base/Base.hpp

```diff
@@ -1,9 +1,12 @@
 #pragma once
 
 
+// forward declarations
 namespace libdnf {
 class Base;
 }  // namespace libdnf
+// lukash: this is really bad. By doing this, you can never be sure which headers can compile standalone, because in our codebase they will have the forward declaration, which they may not have when used elsewhere.
+// lukash: ideally we should get rid of the circular includes by proper design :)
 
```

Comment:
```
-// lukash: this is really bad. By doing this, you can never be sure which headers can compile standalone, because in our codebase they will have the forward declaration, which they may not have when used elsewhere.
-// lukash: ideally we should get rid of the circular includes by proper design :)
-
+// lukash: this is really bad. By doing this, you can never be sure which headers can compile
+// standalone, because in our codebase they will have the forward declaration, which they may not
+// have when used elsewhere. lukash: ideally we should get rid of the circular includes by proper
+// design :)
```